server/metrics/metrics.go

```diff
@@ -1538,6 +1543,50 @@ var (
 		CompressionType,
 	})
 
+	ServerUploadSizeBytes = promauto.NewHistogramVec(prometheus.HistogramOpts{
+		Namespace: bbNamespace,
+		Subsystem: "server",
+		Name:      "upload_size_bytes",
+		Buckets:   prometheus.ExponentialBuckets(1, 10, 9),
+		Help:      "Number of bytes uploaded to the server in each upload. Use the **`_sum`** suffix to get the total uploaded bytes and the **`_count`** suffix to get the number of uploaded files.",
+	}, []string{
+		CacheTypeLabel,
+		ServerName,
+	})
+
+	ServerDownloadSizeBytes = promauto.NewHistogramVec(prometheus.HistogramOpts{
+		Namespace: bbNamespace,
+		Subsystem: "server",
+		Name:      "download_size_bytes",
+		Buckets:   prometheus.ExponentialBuckets(1, 10, 9),
+		Help:      "Number of bytes downloaded from the server in each download. Use the **`_sum`** suffix to get the total downloaded bytes and the **`_count`** suffix to get the number of downloaded files.",
+	}, []string{
+		CacheTypeLabel,
+		ServerName,
+	})
+
+	DigestUploadSizeBytes = promauto.NewHistogramVec(prometheus.HistogramOpts{
```

Comment:
Is the digest upload/download size interesting? Seems like we care about the actual bytes transferred.

Reply:
Not normally but I want to sanity check that compression is working. I will remove these digest metrics afterwards